src/MiddleEnd/ControlFlowGraph/DisasmCFG.fs

```diff
@@ -308,3 +335,10 @@ and private TemporaryDisasmVertex =
 
 /// Mapping from address to TemporaryDisasmVertex.
 and private DisasmVMap = Dictionary<Addr, TemporaryDisasmVertex>
+
+/// Represents the vertex connectivity information.
+and private VertexConnectivity =
+  /// Mergable with the given vertex.
+  | Mergable of IVertex<LowUIRBasicBlock>
+  /// Not mergable, but connectable.
```

Comment:
Please provide an example here and above for future references.

Reply:
Fixed!